src/main/java/org/sonarsource/pluginpackaging/AbstractSonarMojo.java

```diff
@@ -225,6 +228,11 @@ protected String getJreMinVersion() {
     return jreMinVersion;
   }
 
+  @CheckForNull
+  protected String getNodeJsMinVersion() {
```

Comment:
I usually prefer to return an optional in such cases, it's more explicit than the annotation IMO. But it's coherent with the rest of the class